src/types/channels/privacy_level.ts

```diff
@@ -0,0 +1,7 @@
+// TODO: add resource link
+export enum PrivacyLevel {
+  /** The Stage instance is visible publicly, such as on Stage discovery */
+  Public,
```

Comment:
enums start at 0 but discord at 1
```suggestion
  Public = 1,
```